doc/Language/modules.pod6

```diff
@@ -520,9 +520,9 @@ export RAKULIB=/path/to/my-modules,/path/to/more/modules
 
 Note that the comma (',') is used as the directory separator.
 
-The include path will be searched recursively for any modules when Rakudo is
-started. Directories that start with a dot are ignored and symlinks are
-followed.
+The directories in the C<RAKULIB> path will be searched for modules when
```

Comment:
`RAKULIB` is a environment variable, which holds several paths at the same time, so maybe change path by "variable"? Or environment variable?